packages/terra-form-select/src/_MenuUtil.js

```diff
@@ -221,7 +221,7 @@ class MenuUtil {
       return selected === undefined ? options[0].props.value : selected.props.value;
     } else if (searchValue !== state.searchValue) {
       return options[0].props.value;
-    } else if (MenuUtil.findByValue(options, active)) {
+    } else if (active !== null && MenuUtil.findByValue(options, active)) {
```

Comment:
Curious, is it possible for active to ever be undefined?

Reply:
No, it shouldn't be possible. We also can't do `active &&` or a variation of `!active` because an empty string is a valid value and evaluates as false.